server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/model/OnlineSchemaChangeTaskResult.java

```diff
@@ -30,5 +30,5 @@
 @NoArgsConstructor
 @EqualsAndHashCode(callSuper = true)
 public class OnlineSchemaChangeTaskResult extends AbstractFlowTaskResult {
-    private List<ScheduleTask> tasks;
+    private List<ScheduleTaskEntity> tasks;
```

Comment:
Entity class is for persistent layer, may not directly referenced from DTO class

Reply:
look at my fix comment. ScheduleTask is not designed to deserialize from String. cause  ScheduleTaskParameters is interface and no implementation provided. 
A better solution is  use template or provided a deserialize function that may have a lot duplicate code with ScheduleTaskMapper.entityToParameters(). Both solution needs a lots code change. 
This is patch is provided meanly for concern of stability